readthedocs/integrations/models.py

```diff
@@ -332,6 +350,24 @@ def can_sync(self):
             return False
 
 
+class GitHubApp(Integration):
+    integration_type_id = Integration.GITHUBAPP
+    has_sync = False
+    is_remote_only = True
+
+    class Meta:
+        proxy = True
+
+    def get_absolute_url(self):
+        # TODO how to get the URL of the project GHA view?
+        return self.provider_data.get("url", "https://example.com")
```

Comment:
We are generating this URL in https://github.com/readthedocs/readthedocs.org/blob/humitos%2Fexttheme-pypackage/readthedocs/oauth/migrate.py#L41-L58

Reply:
That seems to be the install page URL though, ie:

https://github.com/apps/read-the-docs-business/installations/new

`../installations/new/permissions` 404s for me, but I get this on the `/new/` endpoint:

![image](https://github.com/user-attachments/assets/67bedc11-d8bf-42da-b788-b8f177cc2ec5)

I thought we'd want to link into the specific app installation that affects the repository, like:

https://github.com/organizations/readthedocs/settings/installations/65539825

![image](https://github.com/user-attachments/assets/20beef69-b801-4ed7-9c1e-5f2da9b683d4)

I don't know yet how we get this URL though, especially because it could come from a user or an organization.

@stsewd can you clarify what URL we want here and how we're generating it?